cdk/lib/dotcom-components.ts

```diff
@@ -1,79 +1,93 @@
-import { ComparisonOperator, Metric, TreatMissingData } from '@aws-cdk/aws-cloudwatch';
-import type { Policy } from '@aws-cdk/aws-iam';
-import type { App } from '@aws-cdk/core';
-import { Duration } from '@aws-cdk/core';
-import { AccessScope, GuEc2App } from '@guardian/cdk';
-import { Stage } from '@guardian/cdk/lib/constants/stage';
+import { GuEc2App } from '@guardian/cdk';
```

Comment:
in this file it's mostly changes to imports, but the indentation changed :(

Reply:
Hiding whitespace in the diff view helped with this :-) 
![Screenshot 2023-03-03 at 14 34 43](https://user-images.githubusercontent.com/379839/222747617-3c394cd0-2a70-4a77-bd3a-b2ccade2ba44.png)